.travis.yml

```diff
@@ -156,21 +156,31 @@ script:
   # module is only found if the test directory is in the PYTHONPATH.
   # - export PYTHONPATH="$PYTHONPATH:./ci/"
 
+  CHANGED_FILES_STR=($(git diff --name-only $TRAVIS_COMMIT_RANGE))
```

Comment:
It'll be important to short circuit even before compiling Ray (in situations where we can do that).